UnitySDK/Assets/ML-Agents/Examples/Crawler/Meshes/CrawlerCourt.fbx.meta

```diff
@@ -0,0 +1,99 @@
+fileFormatVersion: 2
```

Comment:
What is the difference between a Mesh and a Prefab? Why do we have two separate folders now?

Reply:
mesh is the raw fbx file normally. prefab is the unity asset file

Reply:
Why do we need separate folders? Why do we have meshes for the Crawler platform instead of a prefab? The platform is a bunch of cubes, why create a mesh for it?

Reply:
some platforms use a custom mesh made in an external 3d modeling program. the fbx files for those items are put in a folder called "meshes"

Reply:
since the 3d model file is different than a unity prefab file they should live in a different folder called "Meshes"